README.md

```diff
@@ -49,6 +49,10 @@ is as easy as sending a text message to your client!
     - [CorporateBalance](#get-your-corporatebalance): View your corporate balance
     - [CorporateTransactions](#query-corporatetransactions): View the transactions that have affected your corporate balance
     - [CorporateEnums](#corporate-enums): Query enums related to the corporate purchases, such as merchant categories, countries and card purchase methods
+    - [MerchantCard](#query-merchantcard): The Merchant Card resource stores information about cards used in approved purchases. These cards can be used in new purchases without the need to create a new session.
+    - [MerchantSession](#create-a-merchantsession): The Merchant Session allows you to create a session prior to a purchase. Sessions are essential for defining the parameters of a purchase, including funding type, expiration, 3DS, and more.
+    - [MerchantPurchase](#create-a-merchantpurchase): The Merchant Purchase resource can be used to charge customers with credit or debit cards. If a card hasn't been used before, a Merchant Session Purchase must be created and approved with that specific card before it can be used directly in a Merchant Purchase.
+    - [MerchantInstallment](#query-merchantinstallment): Merchant Installments are created for every installment in a purchase. These resources will track its own due payment date and settlement lifecycle.
```

Comment:
Deixa essas descrições mais breves, que nem os outros lugares.
Exemplo pra MerchantSession: "Manage a session to run a purchase"